python/src/aac/plugins/first_party/specifications/specifications.yaml

```
@@ -34,6 +34,14 @@ ext:
       - name: spec
         type: Specification
 ---
+ext:
+  name: addSpecTraceToModel
+  type: Model
```

This fixed it for me. This is something I'd like to address by developing an AaC coding style and then standardizing the core spec and plugins.
```suggestion
  type: model
```